code/scripts/spawn.bash

```diff
@@ -62,7 +62,7 @@ export MALACHITE__TEST__VOTE_EXTENSIONS__ENABLED="false"
 export MALACHITE__TEST__VOTE_EXTENSIONS__SIZE="1KiB"
 
 echo "Compiling '$APP_BINARY'..."
-cargo build -p $APP_BINARY
+cargo build -p $APP_BINARY --release
```

Comment:
We usually do not need to run in release mode for manual tests

```suggestion
cargo build -p $APP_BINARY
```

Moreover, because the release flag is missing in the `cargo run` invocation, we are actually always running the debug version.